aws/vpc-public-only/README.md

```diff
@@ -0,0 +1,38 @@
+# aws/vpc-public-only - Virtual Private Cloud
+This module is used to create a VPC along with the necessary configuration to
+be useful.
+
+## What this does
+
+ - Create VPC named after `app_name` and `app_env`
+ - Create public subnet for each `aws_zones` specified
+ - Provision a Internet Gateway and configure public subnets to route through it
+ - Create a DB subnet group including all public subnets
+
+## Required Inputs
+
+ - `app_name` - Name of application, ex: Doorman, IdP, etc.
+ - `app_env` - Name of environment, ex: production, testing, etc.
```

Comment:
Do you want to shorten the suggested `app_env` values? (e.g. "prod, test, etc.")

Reply:
sure